deflect/Segment.h

```diff
@@ -56,6 +56,8 @@ struct Segment
     /** Parameters of the segment. */
     SegmentParameters parameters;
 
+    View view = View::mono; //!< Eye pass for the segment
```

Comment:
It's just a style remark but all other 3 fields in this class use the /** ... */ syntax, and this one in the middle has a //!< ... on the side. The //!< syntax is more compact, but then I would expect all fields to follow the same style... but it's not a big deal ;-)